modules/react/button/lib/TertiaryButton.tsx

```diff
@@ -173,16 +173,14 @@ export const TertiaryButton = createComponent('button')({
         icon={icon}
         iconPosition={iconPosition}
         size={size}
-        cs={[
+        {...mergeStyles(elemProps, [
```

Comment:
curious in secondary we remove the props iconPositon and size and just pass it as elemProps, do we wanna do the same here?

Reply:
No, because `iconPosition` and `size` are used inside this render function. `size` has a default being set in the deconstructed props and `iconPosition` is used in `baseIconPosition` on line 161.